src/parser/priest/shadow/modules/talents/VoidTorrent.tsx

```diff
@@ -37,33 +42,56 @@ class VoidTorrent extends Analyzer {
     return Object.keys(this._voidTorrents).map(key => this._voidTorrents[key]);
   }
 
-  get totalWasted() {
+  get timeWasted() {
     return this.voidTorrents.reduce((total, c) => total + c.wastedTime, 0) / 1000;
   }
 
-  get suggestionThresholds() {
+  get insanityWasted() {
+    return this.timeWasted * VOID_TORRENT_INSANITY_PER_SECOND;
+  }
+
+  get insanityOvercapped() {
+    // Can't use the Energize event's waste because it shows 0 for some reason even if you overcap, so we calculate based on the time channeling vs. how much sanity you should've gotten from it
+    return ((this.totalChannelingTime / 1000) * VOID_TORRENT_INSANITY_PER_SECOND) - this.insanityGained;
+  }
+
+  get interruptThreshold() {
     return {
-      actual: this.totalWasted,
+      actual: this.timeWasted,
```

Comment:
So for this threshold and the one below it. You should try to avoid using thresholds based on the sum of something over the course of the fight whenever possible.

What i mean by that, is say you make 5 mistakes in a fight, dont set the thresholds to 5 mistakes, 10 mistakes, 15 mistakes, instead use something like a percentage of mistakes, or mistakes per minute, or something like that.

the reason is because on a short fight, it will be a lot harder to trigger the advice, and on a long fight it will be a lot easier to trigger it. 

That said, some things are like .. this should NEVER happen and then it is ok, but if its something like it can happen but minimize it ... then you want to use different thresholds.

Reply:
That makes sense, the values sounded good on paper but a 3 min fight vs 10 min fight would make a huge difference. Have updated it to use percentage calculations.